recipes/fgpyo/meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
   host:
     - pip
     - python >=3.8
-    - poetry
+    - hatchling
```

Comment:
see: update to uv, which uses hatchling: https://github.com/fulcrumgenomics/fgpyo/blame/dc6f9556b106a2f84c29984c682312bf1a1214fa/pyproject.toml#L75-L76